share/chruby/chruby.sh

```diff
@@ -42,6 +42,8 @@ function chruby_use()
 	export RUBY_ROOT="$1"
 	export RUBYOPT="$2"
 	export PATH="$RUBY_ROOT/bin:$PATH"
+	local ruby_name
+	ruby_name=$(basename "$RUBY_ROOT")
```

Comment:
These two lines can be shortened into a single line.

Reply:
Also `${RUBY_ROOT##*/}` will be more efficient then invoking `$(basename ...)`.

Reply:
RIght. The reason for the two lines was to avoid a ShellCheck warning:
```
In share/chruby/chruby.sh line 45:
	local ruby_name=$(basename "$RUBY_ROOT")
              ^-------^ SC2155: Declare and assign separately to avoid masking return values.
```
But `${RUBY_ROOT##*/}` avoids that altogether so I'll just use that.

Reply:
Done.